tools/pipelines/templates/upload-dev-manifest.yml

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+# Licensed under the MIT License.
+
+parameters:  
+- name: buildDirectory
+  type: string
+  
+- name: STORAGE_ACCOUNT
+  type: string
+
+- name: STORAGE_KEY
+  type: string
+
+steps:
+- task: Bash@3
+  displayName: Generate release reports
+  inputs:
+    targetType: 'inline'
+    workingDirectory: ${{ parameters.buildDirectory }}
+    script: |
+      mkdir generate_release_reports
+      flub release report -g client -o generate_release_reports --baseFileName manifest
+
+- task: Bash@3
+  displayName: Update release report version
+  inputs:
+    targetType: 'inline'
+    workingDirectory: ${{ parameters.buildDirectory }}
+    script: |
+      mkdir upload_release_reports
+      flub release report-unreleased --version $(SetVersion.version) --fullReportFilePath generate_release_reports/manifest.full.json --outDir upload_release_reports --branchName '$(Build.SourceBranch)'
+
+- task: PublishPipelineArtifact@1
+  displayName: Publish release reports
+  continueOnError: true
+  inputs:
+    targetPath: ${{ parameters.buildDirectory }}/upload_release_reports
+    artifactName: 'release_reports'
```

Comment:
rc1 was before we started using 1ES pipelines so this should be ok in this port, compared to the rc2 and rc3 ones.